core/src/main/java/org/renjin/sexp/Environment.java

```diff
@@ -257,14 +259,26 @@ public void setVariableUnsafe(String name, SEXP value) {
   }
 
   public void setVariable(Context context, String name, SEXP value) {
+    /*

```

Comment:
This belongs in the method Javadocs. Please review http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html as a good resource on writing good API documentation.